src/components/organisms/category/Board.jsx

```diff
@@ -1,60 +1,72 @@
-import { useEffect, useState } from 'react';
-import { Link, useLocation } from 'react-router-dom';
+import { useState, useEffect } from 'react';
+import { Link, useLocation, useSearchParams, useParams } from 'react-router-dom';
 import styled from 'styled-components';
-import { ArticleService } from '../../../network';
-import { ArticleList, Body, Wrapper } from '../../atoms/Board';
-import { PageSelector } from './';
-import BoardHeader from './BoardHeader';
+import BoardHeader from '@common/Board/BoardHeader';
+import { ArticleList, Body, Wrapper } from '@components/atoms/Board';
+import { serializeFormQuery } from '@root/common/utils';
+import { getArticles } from '@common/hooks/api/article';
+import { getSearchResults } from '@common/hooks/api/search';
 import PreviewArticle from './PreviewArticle';
+import PageSelector from './PageSelector';
+import { getCategory, getCategoryName } from '@root/common/hooks/api/category';
```

Comment:
```suggestion
import { serializeFormQuery } from '@root/common/utils';
import { getCategory, getCategoryName } from '@root/common/hooks/api/category';
import { getArticles } from '@common/hooks/api/article';
import { getSearchResults } from '@common/hooks/api/search';
import PreviewArticle from './PreviewArticle';
import PageSelector from './PageSelector';
```

임포트할때 `@root` 경로에서 불러오는 컴포넌트나 함수끼리 같이 배치하고
`@common` 경로에서 불러오는 컴포넌트나 함수끼리 같이 배치하면 좋을것같아요